app/src/main/res/layout/activity_main.xml

```diff
@@ -52,4 +52,4 @@
         app:headerLayout="@layout/nav_header_main"
         app:menu="@menu/activity_main_drawer" />
 
-</android.support.v4.widget.DrawerLayout>
+</android.support.v4.widget.DrawerLayout>
```

Comment:
there is no change in this file. Remove this file from here.

Reply:
how ?